### DISCUSSION TITLE
Pull Request Environments for Saleor devs/maintainers

### DISCUSSION THREAD

**Discussion by waveywaves** [saleor/saleor#10600] [Deployment]
I work at Uffizzi https://uffizzi.com/ as an Open Source Integrations Engineer. I am working on getting PR Environments for Saleor. I want to know how would the maintainers like these preview envs to function. What are the features they would like to see ?

**Comment by peelar** [comment]
Hello @waveywaves 👋 

Thanks for reaching out. Can you please describe what a PR environment is and what would be the advantages of using it?

**Comment by waveywaves** [comment]
Hey @peelar :wave: 

Hope you are doing well ! 
Pull Request Environment / Preview would be a ephemeral deployment of pull requests on your repo. This preview helps reviewers to see the code changes in action against other components. These previews would be full stack in nature. eg: If Pull Request Envs/Previews are enabled for a frontend repo, the frontend code would be built and previewed against a backend and database. It reduces effort taken by the reviewer to pull, build, and locally test the code heavily.

**Reply by peelar** [reply]
Thanks for the explanation. We currently have such a feature and are quite happy with it. If that ever changes, we will know what door to knock on 😄.

**Reply by waveywaves** [reply]
We want to provide this to Saleor with 100K+ preview minute free ! What solution are you using currently ? Does it run previews on each PR ?
